src/topology/compact_open.lean

```diff
@@ -57,32 +58,34 @@ topological_space.generate_open.basic _ (by dsimp [mem_set_of_eq]; tauto)
 
 section functorial
 
-variables {g : β → γ} (hg : continuous g)
+variables (g : C(β, γ))
 
-def induced (f : C(α, β)) : C(α, γ) := ⟨g ∘ f, hg.comp f.continuous⟩
+def induced (f : C(α, β)) : C(α, γ) := g.comp f
```

Comment:
This looks like more than just linting. Can you update the PR description + title, or split this into a separate PR?

Reply:
Split in #9351.